pkg/attestation/crafter/runner.go

```diff
@@ -41,12 +42,21 @@ type SupportedRunner interface {
 
 	// ID returns the runner type
 	ID() schemaapi.CraftingSchema_Runner_RunnerType
+
+	// WorkflowFilePath returns the workflow file path associated with this runner
+	WorkflowFilePath() string
+
+	// IsAuthenticated returns whether the runner is authenticated or not
+	IsAuthenticated() bool
+
+	// RunnerEnvironment returns the runner environment
+	RunnerEnvironment() runners.RunnerEnvironment
```

Comment:
```suggestion
	Environment() runners.RunnerEnvironment
```

since Runner is already the name of the interface